test/v2/config/environment/preprod.json

```diff
@@ -1,5 +1,5 @@
 {
-  "SRSPath": "../../../resources/srs",
```

Comment:
Relative paths are evil. Not sure how this was working though... I'm running this from the Makefile in `test/v2`, so should be `../..` to reach root dir. We should prob use relative paths wrt root though?